config-api/src/main/java/org/keycloak/config/TransactionOptions.java

```diff
@@ -5,17 +5,26 @@
 
 public class TransactionOptions {
 
-    public static final Option<Boolean> TRANSACTION_XA_ENABLED = new OptionBuilder<>("transaction-xa-enabled", Boolean.class)
+    public static final Option<Boolean> DB_XA_ENABLED = new OptionBuilder<>("db-xa-enabled", Boolean.class)
             .category(OptionCategory.TRANSACTION)
-            .description("Manually override the transaction type. Transaction type XA and the appropriate driver is used by default.")
+            .description("Set if the XA transaction type is used. Transaction type XA and the appropriate database driver is used by default.")
```

Comment:
I would change to:

```
If set to false, allow using a non-XA data source in case the database does not support XA transactions. By default, only XA data sources are supported. This option is automatically set to false if the `transactions-jta-enabled` option is set to false. 
```

Reply:
+1, will change

Reply:
decided to remove the part mentioning the hidden var.